src/features/cart/ui/CartDialogContent.svelte

```diff
@@ -74,7 +73,7 @@
     {/each}
     
     <div class="grid place-items-center">
-      <button  on:click= {() => goto('/order')} class="rounded-full border border-lines py-6 mt-7 px-11">Оформити замовлення</button>
+      <a href="/order"> Замовити</a>
```

Comment:
Where is styles? where is `class="rounded-full border border-lines py-6 mt-7 px-11"`